engine/access/rpc/backend/backend.go

```diff
@@ -132,9 +137,32 @@ func New(
 
 	retry.SetBackend(b)
 
+	var err error
+	preferredENIdentifiers, err = enIDsToIndentifierList(preferredExecutionNodeIDs)
+	if err != nil {
+		log.Fatal().Err(err).Msg("failed to convert node id string to Flow Identifier for preferred EN map")
+	}
+
+	fixedENIdentifiers, err = enIDsToIndentifierList(fixedExecutionNodeIDs)
+	if err != nil {
+		log.Fatal().Err(err).Msg("failed to convert node id string to Flow Identifier for fixed EN map")
+	}
+
 	return b
 }
 
+func enIDsToIndentifierList(ids []string) (flow.IdentifierList, error) {
```

Comment:
I would suggest `toIdentifierList(ids []string)(flow.IdentifierList, error)`.the implementation does rely on identifiers belonging to the execution node. Also, the input is not an `ID` in the protocol terminologies.